client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-started/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { StepContainer } from '@automattic/onboarding';
+import React from 'react';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import type { Step } from '../../types';
+
+const SiteMigrationStarted: Step = function () {
```

Comment:
Step already create for the issue https://github.com/Automattic/wp-calypso/issues/91852